frontend/src/components/shared/pick-track/index.tsx

```diff
@@ -122,4 +126,4 @@ class PickTrack extends React.Component<IPickTrackProps, IPickTrackState> {
   }
 }
 
-export default PickTrack;
+export default PickTrack
```

Comment:
@ankita-sri can you re-add the semicolon here? (and amend the commit)

Reply:
did 🗡

Reply:
@ankita-sri squash commits please :P

Reply:
ping @ankita-sri